src/controllers/auth/resetPassword.ts

```diff
@@ -0,0 +1,53 @@
+import { generateRandomString, hashData } from '@/common/utils';
+import AppError from '@/common/utils/appError';
+import { AppResponse } from '@/common/utils/appResponse';
+import { catchAsync } from '@/middlewares';
+import { UserModel as User } from '@/models/userModel';
+import { Request, Response } from 'express';
+import { DateTime } from 'luxon';
+import { addEmailToQueue } from '../../queues/emailQueue';
+
+export const resetPassword = catchAsync(async (req: Request, res: Response) => {
+	const { email } = req.body as { email: string };
```

Comment:
why using as {email: string} ? isn't there a validation to ensure this ?

Reply:
there is i suppose, i guess i just followed what @Chimise did in sign in hehe

Reply:
i have changed this in [this commit](https://github.com/abeg-help/backend/pull/89/commits/1a29bc51d428b489017e3d55ecc86316a3bc7be2`)